pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -18,80 +18,83 @@ Please read this Privacy Policy carefully before accessing or using the Website.
+5. Any student academic data provided by us or by our school district partners to third party evaluators for the purpose of evaluating our courses in meeting our mission will be de-identified (per standard industry practice).
 6. We strive to provide you with access to and control over the information you give us (as detailed below), and we take the protection of your information very seriously.
 7. When student Personal Information is provided to Code.org by a school or school district, Code.org agrees to retain such information as directed by the school or school district.
 
 
 
 ## Privacy Policy
-We are committed to creating a safe and secure environment for learners of all ages on our websites located at [https://code.org](https://code.org), [https://studio.code.org](https://studio.code.org), [https://csedweek.org](https://csedweek.org), [https://hourofcode.com](https://hourofcode.com), [https://curriculum.code.org](https://curriculum.code.org), [https://forum.code.org](https://forum.code.org), other websites, application programming interface (our “API”), and online services (collectively, our “Website”). Visitors and users of the Website and participants in our education programs are referred to individually as “User” and collectively as “Users”. As a not-for-profit, 501(c)(3) organization, we use the data we receive only insofar as it helps our mission of providing a great computer science education for every student who wants it. We established ourselves as a not-for-profit organization so that a for-profit motive will not interfere with our mission of providing a trusted educational resource.
+We are committed to creating a safe and secure environment for learners of all ages on our websites located at [https://code.org](https://code.org), [https://studio.code.org](https://studio.code.org), [https://csedweek.org](https://csedweek.org), [https://hourofcode.com](https://hourofcode.com), [https://curriculum.code.org](https://curriculum.code.org), [https://forum.code.org](https://forum.code.org), other websites, application programming interface (our “API”), and online services (collectively, our “Website”). Visitors and users of the Website and participants in our education programs are referred to individually as “User” and collectively as “Users”. Any User that creates a student account is referred to as “Student” even if they are using the Code.org Website outside of a school setting. Similarly, any User that creates a teacher account is referred to as “Teacher.” As a not-for-profit, 501(c)(3) organization, we use the data we receive only insofar as it helps our mission of providing a great computer science education for every student in every school. We established ourselves as a not-for-profit organization so that a for-profit motive will not interfere with our mission of providing a trusted educational resource.
```

Comment:
Visitors and `Users`?

Reply:
on our `Websites` located

Reply:
other `Websites`, application programming

Reply:
I think those are all being used before defining the capital version, so I think the lowercase is fine in these cases.